src/gc/gcinterface.h

```diff
@@ -526,6 +527,9 @@ class IGCHeap {
     // throughout the VM.
     virtual HRESULT GarbageCollect(int generation = -1, BOOL low_memory_p = FALSE, int mode = collection_blocking) = 0;
 
+    // Gets the smallest GC generation that runs finalizers.
+    virtual unsigned GetMinFinalizingGeneration() = 0;
```

Comment:
I do not understand what this means or why you need it. The finalizers run even after gen0 GC:

```
using System;
using System.Runtime.CompilerServices;

class Program
{
    ~Program()
    {
        Console.WriteLine("Here");
    }

    [MethodImpl(MethodImplOptions.NoInlining)]
    static void foo()
    {
        GC.KeepAlive(new Program());
    }

    static void Main(string[] args)
    {
        GC.Collect();
        foo();
        GC.Collect(0);
        for (;;);
    }
}
```